data/lib/core/player.lua

```diff
@@ -158,6 +158,8 @@ function Player.transferMoneyTo(self, target, amount)
 	end
 
 	self:setBankBalance(self:getBankBalance() - amount)
+	-- incase server crashes that we do not duplicate money
+	db.query("UPDATE `players` SET `balance` = " .. self:getBankBalance() .. " WHERE `id` = '" .. self:getGuid() .. "'")
```

Comment:
However, if the target is online, it will be duplicated.

Reply:
The problem is at current stage we cannot fully avoid this unless we remove transfer to offline players or we introduce some kind of function which only saves the transfer money of the offline player once the player who send it was saved aswell, or this will always have the potential to go out of sync when the server crashes

Reply:
https://github.com/otland/forgottenserver/pull/4671/commits/53dbb165f5dce048fe072e4f4f8f58593de9d530
better approach but I don't like the saving player part, as it could introduce more out of sync stuff on crash